### PR TITLE
Added new icons and changed settingspanel page style

### PR DESCRIPTION
**Pull request by alesmaye**
## Description

Copied css classes from existing skin to make the new skin.
Added the currently available icons
Made the new settings panel page style
IMPORTANT: `UIContainer` (in the UIFactory) now needs the `cssClasses` argument necessarily because there is no more "standard skin" (this is because there are now two skins)

## Checklist (for PR submitter and reviewers)

- [x] `CHANGELOG` entry

### CODE REVIEW

File: src/scss/skin-super-modern/_skin-modern-smallscreen.scss [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  }
+
+  // Adjustments for screen width x <= 600
+  // sass-lint:disable no-empty-rulesets
+  &.#{$prefix}-layout-max-width-400,
+  &.#{$prefix}-layout-max-width-600 {
+    // none yet
+  }
+
+  // IE9 compatibility: fallback for missing flexbox support
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/_skin-modern-smallscreen.scss]: I don't think we need to support IE9 anymore :)

File: src/scss/skin-super-modern/components/_audiotracksettingspaneltogglebutton.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// demo for extracted audio tracks and subtitle settings from the settings panel direct into the controlBar
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_audiotracksettingspaneltogglebutton.scss]: Is this component even used in the super-modern-ui?

File: src/scss/skin-super-modern/components/_clickoverlay.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+@import '../variables';
+
+%ui-clickoverlay {
+  @extend %ui-button;
+}
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_clickoverlay.scss]: Do we actually need this? I mean if it just extends the ui-button, everything that extends/uses this may very well just extend/use the ui-button directly?

File: src/scss/skin-super-modern/components/_component.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+@import '../variables';
+
+%ui-component {
+  /*! placeholder to avoid removal of empty selector */
+  //outline: 1px solid red;
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_component.scss]: this commented code can be removed.

File: src/scss/skin-super-modern/components/_controlbar.scss [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+          display: table-cell;
+          vertical-align: middle;
+        }
+
+        .#{$prefix}-ui-volumeslider {
+          width: 10%;
+        }
+      }
+    }
+  }
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_controlbar.scss]: We should again be able to remove this.

File: src/scss/skin-super-modern/components/_errormessageoverlay.scss [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    position: absolute;
+    right: 3em;
+    text-align: center;
+    user-select: text;
+    white-space: pre-line; // enable linebreak in text
+
+    // Vertically center the label
+    & {
+      // sass-lint:disable no-vendor-prefixes
+      -ms-transform: translateY(-50%); // required for IE9
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_errormessageoverlay.scss]: can be remove

File: src/scss/skin-super-modern/components/_seekbar.scss [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+  position: relative;
+  width: 100%;
+
+  $bar-inset: auto;
+
+  .#{$prefix}-seekbar {
+
+    %bar {
+      // sass-lint:disable no-vendor-prefixes
+      -ms-transform-origin: 0 0; // required for IE9
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_seekbar.scss]: can be removed again

File: src/scss/skin-super-modern/components/_seekbar.scss [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+      .#{$prefix}-seekbar-bufferlevel,
+      .#{$prefix}-seekbar-playbackposition {
+        transition: none;
+      }
+    }
+  }
+
+  &.#{$prefix}-vertical {
+    .#{$prefix}-seekbar {
+      //overflow-y: hidden;
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_seekbar.scss]: Can be removed completely

File: src/scss/skin-super-modern/components/_seekbar.scss [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+      height: 100%;
+      width: auto;
+
+      %bar-vertical {
+        right: 0;
+        top: auto;
+      }
+
+      %positionmarker-vertical {
+        //@extend %positionmarker-horizontal;
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_seekbar.scss]: Can also be removed

File: src/scss/skin-super-modern/components/_settingspanel.scss [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+  @include hidden-animated-with-additional-transitions($animation-duration,
+    (
+      height: (.35s, cubic-bezier(.4, 0, .2, 1)),
+      width: (.35s, cubic-bezier(.4, 0, .2, 1))
+    )
+  );
+
+  $background-color: transparentize(#212226, .15);
+  //#3e3f40
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_settingspanel.scss]: Is that a color code? Why is it here 😄

File: src/scss/skin-super-modern/components/_settingspanel.scss [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+  overflow-y: auto;
+  bottom: 3.5em;
+  right: 1em;
+
+  > .#{$prefix}-container-wrapper {
+    overflow-y: auto;
+
+    > * {
+      //margin: 0 .5em;
+    }
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_settingspanel.scss]: Why is this no warning 🤔 
Pleae remove this empty child selector.

File: src/scss/skin-super-modern/components/_settingspanelpage.scss [hunk 12]
```diff
@@ -0,0 +1,51 @@
+@import '../variables';
+
+%ui-settings-panel-page {
+  display: none;
+  //width: 100%;
+  //min-width: 10px;
+  //overflow: hidden;
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_settingspanelpage.scss]: No commented out code please

File: src/scss/skin-super-modern/components/_settingspanelpage.scss [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+
+  &.#{$prefix}-active {
+    display: block;
+  }
+
+  // A "line" in the panel: a container holding a label + control
+  .#{$prefix}-container-wrapper > * {
+    // Labels
+    //width: 100%;
+    //overflow: hidden;
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_settingspanelpage.scss]: Again, please remove this.

File: src/scss/skin-super-modern/components/_settingspanelpage.scss [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+    // Controls (e.g. selectbox)
+    &.#{$prefix}-ui-selectbox {
+      margin-left: 10%;
+      width: 45%;
+    }
+  }
+
+  .#{$prefix}-ui-settings-panel-item {
+    //border-bottom: 1px solid transparentize($color-secondary, .7);
+    //padding: .2em 0;
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_settingspanelpage.scss]: Again again, please remove this.

File: src/scss/skin-super-modern/components/_settingspanelpage.scss [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+
+    &:hover {
+      background-color: transparentize(#54565a, .15);;
+    }
+
+    &.#{$prefix}-hidden {
+      display: none;
+    }
+  }
+  //border: green solid thin;
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_settingspanelpage.scss]: For the last time in this file :) Please remove

File: src/scss/skin-super-modern/components/_settingspanelpagebackbutton.scss [hunk 16]
```diff
@@ -0,0 +1,30 @@
...
+      content: '';
+      height: .6em;
+      margin-left: -.8em;
+      position: absolute;
+      top: .6em;
+      transform: rotate(45deg);
+      width: .6em;
+    }
+  }
+  //border: blue solid thin;
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_settingspanelpagebackbutton.scss]: I guess this was useful for debugging, but please remove it.

File: src/scss/skin-super-modern/components/_uicontainer.scss [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+        box-shadow: inset -4px -3px 2px 9px $color-focus;
+      }
+
+      &:focus:not(.#{$prefix}-focus-visible) {
+        box-shadow: none;
+      }
+    }
+  }
+
+  // IE9 compatibility: set transparent 1x1 pixel png background image to make it capture mouse events (IE9 does not capture events in areas without image or color content)
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_uicontainer.scss]: This can also go.

**Review by bitmovin-kenny — CHANGES_REQUESTED** [review]
Found some debugging artifacts and some weird stuff which was most likely copied. Nice work so far 💪 
Please also fix all the linter problems in the new files.

File: src/scss/skin-super-modern/components/_audiotracksettingspaneltogglebutton.scss (hunk 2 above)
Comment (alesmaye) [reply src/scss/skin-super-modern/components/_audiotracksettingspaneltogglebutton.scss]: no, already removed

File: src/scss/skin-super-modern/components/_errormessageoverlay.scss [hunk 18]
```diff
@@ -23,7 +23,6 @@
     // Vertically center the label
     & {
       // sass-lint:disable no-vendor-prefixes
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_errormessageoverlay.scss]: this comment is also no longer needed

File: src/scss/skin-super-modern/components/_seekbar.scss [hunk 19]
```diff
@@ -21,7 +21,6 @@ $seekbar-height: .3125em;
 
     %bar {
       // sass-lint:disable no-vendor-prefixes
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_seekbar.scss]: This is also not needed anymore

File: src/scss/skin-super-modern/components/_settingspanelpage.scss [hunk 20]
```diff
@@ -1,20 +1,12 @@
 @import '../variables';
 
 %ui-settings-panel-page {
-  display: none;
```
Comment (bitmovin-kenny) [inline src/scss/skin-super-modern/components/_settingspanelpage.scss]: Was this deleted by accident?

**Review by bitmovin-kenny — CHANGES_REQUESTED** [review]
Much nicer without all these warnings 💪 
Just some minor nitpicks left

File: src/scss/skin-super-modern/components/_settingspanelpage.scss (hunk 20 above)
Comment (alesmaye) [reply src/scss/skin-super-modern/components/_settingspanelpage.scss]: Yes, was deleted accidentally

**Review by bitmovin-kenny — APPROVED** [review]
LGTM, please wait for the review of @stonko1994

File: src/scss/skin-super-modern/_mixins.scss [hunk 21]
```diff
@@ -137,10 +137,10 @@
   &:focus {
     box-shadow: $focus-element-box-shadow;
     outline: none;
-  }
 
-  &:focus:not(.#{$prefix}-focus-visible) {
-    box-shadow: none;
-    outline: none;
+    &:not(.#{$prefix}-focus-visible) {
```
Comment (stonko1994) [inline src/scss/skin-super-modern/_mixins.scss:141]: Why did you remove this `focus`? 🤔

**Review by stonko1994 — COMMENTED** [review]
When I'm trying to use the `superModernMobileUI` I do not see any UI 🤔 Is this expected at this point?

File: src/scss/skin-super-modern/_mixins.scss (hunk 21 above)
Comment (alesmaye) [reply src/scss/skin-super-modern/_mixins.scss:141]: It is still there. The `:not` just nested in the `&:focus`. Otherwise the linter would throw a warning.

```
&:focus {
    box-shadow: $focus-element-box-shadow;
    outline: none;

    &:not(.#{$prefix}-focus-visible) {
      box-shadow: none;
      outline: none;
    }
  }

```

**Review by stonko1994 — APPROVED** [review]

File: src/scss/skin-super-modern/_mixins.scss (hunk 21 above)
Comment (stonko1994) [reply src/scss/skin-super-modern/_mixins.scss:141]: Sorry missed that 🤦
